### PR TITLE
Upgrade ktlint

### PR DESCRIPTION
**Pull request by JcMinarro**
Upgrade KtLint version to a version that requires Traling Commas to avoid PR conflicts

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Apply new format to the code
+c8ef601925956c98b052034cb0f38cffc88e0aae
```
Comment (JcMinarro) [inline .git-blame-ignore-revs:2]: This file allows us to skip any commit from git-blame that doesn't apport anything to the history.
In that case, this commit is the one where the check style has been updated, but the code has no feature change.
[Here](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view) you can read docs about it.

We shouldn't squash this PR to be sure the commit SHA1 doesn't change

**Review by vjgarciag96 — APPROVED** [review]
Nice, thank you!

**Comment by vjgarciag96** [comment]
I think this now has conflicts with the PR to open handlers that got merged just now.

**Comment by JcMinarro** [comment]
I will update it

**Review by vjgarciag96 — APPROVED** [review]

**Pull request merged**